### PR TITLE
UML-51 Pull containers from ECR, Deploy to ECS

### PR DESCRIPTION
**Pull request by TomTucka**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -80,4 +83,11 @@ orbs:
...
-                terraform apply --auto-approve
\ No newline at end of file
+                terraform apply --auto-approve
+      test_view_service_development:
+        executor: terraform
+        steps:
+          - run:
+              name: check response from view service
+              command: |
+                curl https://view.dev.use-an-lpa.opg.service.justice.gov.uk --fail
```
Comment (damacus) [inline .circleci/config.yml]: why no EOL character?

Comment (TomTucka) [reply .circleci/config.yml]: Sorted 💯

File: .circleci/config.yml [hunk 2]
```diff
@@ -80,4 +89,25 @@ orbs:
...
+              name: check response from viewer service
+              command: |
+                curl https://viewer.preprod.use-an-lpa.opg.service.justice.gov.uk --fail
+      test_viewer_service_production:
+        executor: terraform
+        steps:
+          - run:
+              name: check response from viewer service
+              command: |
+                curl https://viewer.use-an-lpa.opg.service.justice.gov.uk --fail
```
Comment (damacus) [inline .circleci/config.yml]: this can go on the line above rather than using `|`

Comment (TomTucka) [reply .circleci/config.yml]: Sorted 💯

File: terraform/s3.tf [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      "${aws_s3_bucket.access_log.arn}",
+      "${aws_s3_bucket.access_log.arn}/*",
+    ]
+
+    effect  = "Allow"
+    actions = ["s3:PutObject"]
+
+    principals {
+      // AWS docs, Account ID that corresponds to the region for your load balancer and bucket.
+      // https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-access-logs.html
```
Comment (damacus) [inline terraform/s3.tf]: Do we still need these docs?
//c @sixdaysandy

Comment (TomTucka) [reply terraform/s3.tf]: Done 💯

**Pull request merged**